packages/panzer/disc-fe/src/Panzer_IntegrationRule.cpp

```diff
@@ -74,23 +74,29 @@ IntegrationRule(const panzer::CellData& cell_data, const std::string & in_cv_typ
   // TODO: This requirement (on arbitrary cubature order) will be dropped with the new Workset design (using descriptors to find integration rules)
   // TODO: These isSide conditions shouldn't be required... I'm missing something...
   if(in_cv_type == "volume"){
-    if(cell_data.isSide()){
-      IntegrationDescriptor::setup(75, IntegrationDescriptor::CV_VOLUME,cell_data.side());
-    } else {
+    TEUCHOS_TEST_FOR_EXCEPT_MSG(cell_data.isSide(),
+                                "IntegrationRule::IntegrationRule : Control Volume 'volume' type requested, but CellData is setup for sides.");
+//    if(cell_data.isSide()){
+//      IntegrationDescriptor::setup(75, IntegrationDescriptor::CV_VOLUME,cell_data.side());
+//    } else {
```

Comment:
This block and a few below are commented out. Have we lost capability here or is it just not needed. Should these be removed?

Reply:
Done